File: Benchmarking_Languages_Project/Scala_Tests/Test7.scala
var timing = DenseVector.zeros[Long](15)
for(i <- 0 until 15){
	
	val start = System.currentTimeMillis()
	val T7 = DenseMatrix.randn(640,640)
	val T7_1 = eig(T7)
	val finish = System.currentTimeMillis()
	timing(i) = (finish - start)
}

println(mean(timing)/1000)

File: Benchmarking_Languages_Project/Python_Tests/Test7.py
from scipy import *
from numpy import *
from time import *

timing = zeros(15)
for i in range(15):
	time1 = time()
	T7 = matrix(random.normal(size=(640,640)))
	T7_1 = linalg.eig(T7)
	timing[i]=time()-time1
print mean(timing)
